lib/parser/ruby27.y

```diff
@@ -319,9 +319,10 @@ rule
                       method_call = @builder.call_method(nil, nil, val[0],
                                         nil, val[1], nil)
 
-                      begin_t, args, body, end_t = val[2]
+                      begin_t, args, max_numparam, body, end_t = val[2]
```

Comment:
@whitequark Any thoughts on this? We flush `@lexer.max_numparam` when the body gets reduced and so in the `block` rule there's no way to get it. That's why I had to pass it down via `result`. Do you see any other way to store this value?

Reply:
Maybe let's create a "dummy" node `numbered_args` so `args` here is either `args` or `numbered_args`. Later builder can use it to emit `block` or `numblock`. This is basically similar to `ident` node that never appears in the final output.

Reply:
Sorry for the delay, this fell between the cracks. I agree with `numbered_args` plan. (Only it should be called `numargs` to keep our convention.)

Reply:
Done, PTAL when you have time